src/Evolution/Systems/CurvedScalarWave/Worldtube/Tags.hpp

```diff
@@ -68,6 +74,56 @@ struct ExcisionSphere : db::SimpleTag {
   }
 };
 
+/*!
```

Comment:
Wrap both tags in `/// @{` and `/// @}` so the docs apply to both

Reply:
added another commit that does the same for `CenteredFaceCoordinates`